shuttle/templates/driver_check_in/load_driver_check_locations.html

```diff
@@ -71,7 +71,7 @@ <h4>Current status:<span class="badge badge-light break-badge">{{ current_break_
         function check_time() {
             var today = new Date();
             var time = today.getHours() + ":" + today.getMinutes();
-            if (time >= {{ next_time.split(' ')[0]|tojson }}) {
+            if (time >= {{ next_time_converted|tojson }}) {
```

Comment:
Instead of passing over additional variables couldn't you just mod today.getHours() by 12? seems a bit easier and saves us from passing over more than we need.

Reply:
I just got an error while just having the check in tab open

```  File "/Users/bam95899/Projects/Sites/shuttle/shuttle/driver_check_in/__init__.py", line 41, in load_driver_view
    next_time_converted = next_check_in['24_hour_time']
KeyError: '24_hour_time'
```

Reply:
Oh I know why that is. This commit should fix it